### ISSUE TITLE
[SUGGESTION] Ambient Occlusion re-implementation 

### ISSUE THREAD

**Issue by Womble-TC** [tsunamayo/Starship-EVO#2838]
A few weeks ago, AO was removed from the game, since it has issues with asteroids. 
Personally, I think this was a pretty big loss, since AO can really help add some depth and detail to a whole range of interiors, exteriors, or just builds in general. Here are some example of before and after ambient occlusion was removed, I have employed the help of the "cat 'o meter™" to help describe my thoughts:
![image](https://user-images.githubusercontent.com/29961381/91641286-36a4ad80-ea1b-11ea-9471-2cd61f183eff.png)
![image](https://user-images.githubusercontent.com/29961381/91641302-59cf5d00-ea1b-11ea-9967-0fcfecc09781.png)

While the AO may have been buggy on asteroids, I feel at least having the option to turn it on and off (even it it were off by default) would be a great addition, especially for people who want to take nice looking screenshots or videos to show off their builds/the game as a whole. I just think the negatives of AO are greatly outweighed by the visual benefits if gives in most scenarios.

**Comment by Womble-TC** [comment]
Also, congrats on the recent player number increase! It beat a new record today. 
![image](https://user-images.githubusercontent.com/29961381/91641649-ba5f9980-ea1d-11ea-9e20-ccfcd53eddb5.png)

**Comment by ProPeach** [comment]
The old occlusion really was beautiful, I'd love to have it back with an option to toggle it in the graphics options

**Comment by GadyZ** [comment]
I really have to say that it looks very flat without it. I think it's a must have.

**Comment by PopkaAkula** [comment]
Getting the old ambient occlusion back along with some fixes for it would be great, it gives the graphics more depth and makes the environments more immersive.

**Comment by glorbo703** [comment]
how do I enable the cat-o-meter ingame? 🤔

**Comment by CreepoBeepo** [comment]
Yeah ambient occlusion would add a lot to the look of the game. I think it would be nice to have it as a option you can toggle.

**Comment by tsunamayo** [comment]
Yep I need to work on that for sure. I had quite a few issue with it so I preferred to ditch it. But I would also have to work on better setting management, as I was a bit shocked by how many player plays the game with potato computer, so I need to make sure to present a valid first setting. All I want is make cat happy!
